composer.json

```diff
@@ -9,18 +9,31 @@
         }
     ],
     "require": {
-        "php": "^5.6|^7.0",
-        "symfony/symfony": "^2.8|^3.0",
-        "symfony/phpunit-bridge": "^3.2",
-        "symfony/monolog-bundle": "^2.1|^3.0",
+        "php": "^5.6 || ^7.0",
         "doctrine/common": "^2.4",
+        "doctrine/data-fixtures": "^1.0",
         "doctrine/doctrine-bundle": "^1.0",
-        "doctrine/phpcr-odm": "^1.1|^2.0",
         "doctrine/phpcr-bundle": "^1.1",
-        "doctrine/data-fixtures": "^1.0",
+        "doctrine/phpcr-odm": "^1.1 || ^2.0",
+        "friendsofsymfony/jsrouting-bundle": "^1.1",
         "jackalope/jackalope": "^1.1.5",
         "jackalope/jackalope-doctrine-dbal": "^1.1.5",
-        "friendsofsymfony/jsrouting-bundle": "^1.1"
+        "symfony/browser-kit": "^2.8 || ^3.0",
+        "symfony/console": "^2.8 || ^3.0",
+        "symfony/css-selector": "^2.8 || ^3.0",
+        "symfony/debug": "^2.8 || ^3.0",
+        "symfony/dependency-injection": "^2.8 || ^3.0",
+        "symfony/doctrine-bridge": "^2.8 || ^3.0",
+        "symfony/framework-bundle": "^2.8.1 || ^3.0",
+        "symfony/http-foundation": "^2.8 || ^3.0",
+        "symfony/http-kernel": "^2.8 || ^3.0",
+        "symfony/monolog-bundle": "^2.8 || ^3.0",
+        "symfony/process": "^2.8 || ^3.0",
+        "symfony/security-bundle": "^2.8 || ^3.0",
+        "symfony/twig-bundle": "^2.8 || ^3.0"
+    },
+    "require-dev": {
+        "symfony/phpunit-bridge": "^4.0"
```

Comment:
was it actually correct to move this to require-dev? this means when something else uses the testing component (hopefully in *their* require-dev) the bridge must be installed separately.

Reply:
If someone wants to use simple-phpunit it is better that they require it directly. Imagine if you dont do it: if someday simfony decides to rename in a major the executable and you are not requiring it, you will end with broken builds. This happened once with twig on sonata, because twig upgraded to 2.0 breaking many things but no one had this dependency declared

Reply:
okay, that makes sense.